aws-cloudfront-naked-domain-redirect/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable project {
```

Comment:
**[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
variable "project" is declared but not used